toolkit/featureforms/src/androidTest/java/com/arcgismaps/toolkit/featureforms/UtilityNetworkNavigationTests.kt

```diff
@@ -57,7 +55,7 @@ class UtilityNetworkNavigationTests : FeatureFormTestRunner(
         val element = featureForm.elements.first() as? UtilityAssociationsFormElement
         assertThat(element).isNotNull()
         // Wait for the associations to load
-        composeTestRule.waitUntil(timeoutMillis = 20_000,) {
+        composeTestRule.waitUntil(timeoutMillis = 30_000,) {
```

Comment:
Minor warnings on this file:  
```suggestion
        composeTestRule.waitUntil(timeoutMillis = 30_000) {
```